docs/UserGuide.md

```diff
@@ -31,104 +31,249 @@
 
 > **Command Format**
 > * Words in `UPPER_CASE` are the parameters.
-> * Items in `SQUARE_BRACKETS` are optional.
+> * The order of parameters does not matter.
+> * Items enclosed within `[ ]` are optional.
 > * Items with `...` after them can have multiple instances.
-> * The order of parameters is fixed.
+> * The use of commas `,` and full stops `.` is optional and will not affect the command.
+> * Words not enclosed within `" "` are not case-sensitive.
+> * For parameters relating to days, they are context-sensitive. For example, specifying "Fri" sets the date parameter as the next Friday from *today*.
+
+> **Tasks**<br>
+> A task will have a name, and may be categorized as follows:
+> * **Floating Task** has no additional details
+> * **Normal Task** has a deadline
+> * **Task with Allocated Timeslot** has a deadline, starting time and ending time
+
+> **Events**<br>
+> Tasks may act as events in our program. Simply enter the starting and ending time
+> and omit the deadline.
+
+> **Recurring tasks and events**
+> Tasks and events may be set to repeat daily/weekly/monthly,
+> recurring after its deadline or end time.
+
+>> Tasks or events which are exact duplicates are combined into one.
 
 #### Viewing help : `help`
 Format: `help`
 
-> Help is also shown if you enter an incorrect command e.g. `abcd`
- 
-#### Adding a person: `add`
-Adds a person to the address book<br>
-Format: `add NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS [t/TAG]...` 
+> If you enter an invalid command (e.g `abcd`), or enter invalid parameters
+> for commands, mini-help messages would be shown on the output box in the
+> program instead.
 
-> Persons can have any number of tags (including 0)
+#### Create a new task/event: `add`
```

Comment:
do you allow adding multiple tasks at the same time? 
may be you need to make a note of this in the description